src/platforms/apple/common/dsym.mdx

```diff
@@ -6,11 +6,45 @@ title: Uploading Debug Symbols
 
 ## With Bitcode {#dsym-with-bitcode}
 
-If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2) **is enabled** in your project, you will have to upload the dSYM to Sentry **after** it has finished processing in iTunesConnect. We also recommend using the latest Xcode version for submitting your build. The dSYM can be downloaded either with [Fastlane](#bitcode-fastlane) or with [sentry-cli](#bitcode-sentrycli).
+If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2) **is enabled** in your project, dSYM files will only be available **after** builds have finished processing in App Store Connect. We recommend using the latest Xcode version for submitting your build.
+
+Sentry can fetch dSYM files automatically as they become available from App Store Connect when setting up the [App Store Connect](#bitcode-appstore) integration.
+
+Alternatively, the dSYM files can be downloaded either with [Fastlane](#bitcode-fastlane) or manually, and uploaded with [sentry-cli](#bitcode-sentrycli).
+
+### Using App Store Connect {#bitcode-appstore}
+
+When using the App Store Connect integration, Sentry will automatically discover and fetch dSYM files for Bitcode builds as they become available on App Store Connect.
+
+To configure the integration, go to **Project Settings > Debug Files** and add App Store Connect as a new **Custom Repository**.
+
+Setting up App Store Connect currently requires two separate credentials.
+
+**App Store Connect API Key**:
+
+Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_ and _Private Key_.
+
+**iTunes Credentials**:
+
+An iTunes user and password are required in addition to the normal App Store Connect API Key, due to limitations of the App Store Connect API. The App Store Connect API Key alone will only allow detection of new builds, but an iTunes user is required to actually fetch the associated dSYM files.
+
+[Create a new Apple ID](https://support.apple.com/en-us/HT204316) specifically for this integration and provide its credentials. The user needs to have the _"Developer"_ role in the appropriate App Store Organization.
+
+<Alert level="warning">
+
+We highly recommend to create a dedicated iTunes user account for the purpose of this integration, and to **not** use a regular user account.
+
+</Alert>
+
+These credentials will need to be verified by a two-factor authentication token that can optionally be requested via SMS.
```

Comment:
I'm not sure if you mean that it's optional whether the 2FA happens by SMS OR that the 2FA *must* have the SMS option. If you mean the second, then just remove the word "optionally". If you mean the first, then remove everything after "token."

```suggestion
These credentials will need to be verified by a two-factor authentication token that can optionally be requested via SMS.
```

Reply:
By default it will automatically pop-up the 2FA token on any apple device that you have linked to the account, but there is a "send sms" button to send the token via SMS instead. How should I phrase that?

Reply:
Got it. 
```suggestion
These credentials will need to be verified by a two-factor authentication token that you can choose to have sent by SMS.
```